soca/soca_3dvar.yaml

```diff
@@ -117,15 +108,15 @@ cost function:
         ssh_min: 0.0   # value at EQ
         ssh_max: 0.0   # value in Extratropics
         ssh_phi_ex: 20 # lat of transition from extratropics
-        chl_min: 0.003
-        chl_max: 10.0
-        biop_min: 0.0
-        biop_max: 1.0e-6
         cicen_min: 0.05
         cicen_max: 0.05
         hicen_min: 0.1
         hicen_max: 0.1
 
+      - linear variable change name: HorizFiltSOCA
```

Comment:
what's the reason for adding this back @hyunchul386 ?

Reply:
@hyunchul386 can you address this?